examples/HelloTriangle.cpp

```diff
@@ -56,9 +56,16 @@ void initTextures() {
         .SetAllowedUsage(nxt::TextureUsageBit::TransferDst | nxt::TextureUsageBit::Sampled)
         .GetResult();
 
-    sampler = device.CreateSamplerBuilder()
-        .SetFilterMode(nxt::FilterMode::Linear, nxt::FilterMode::Linear, nxt::FilterMode::Linear)
-        .GetResult();
+    {
```

Comment:
Maybe not useful for this struct - but should we generate "fill with default" functions for struct initialization?

I think even constructors, like `nxt::SamplerDescriptor makeSamplerDescriptor()`, would work.

Reply:
I thought about adding default to the generator but was worried that if WebGPU chooses different defaults and we update to follow them, we would have tests and samples break. Adding helper functions to make the default descriptor in NXTHelpers sounds good though. I'll look into it.